charts/kubernetes-agent/values.yaml

```diff
@@ -123,6 +123,13 @@ scriptPods:
   # @section -- Script pod values
   disruptionBudgetEnabled: true
 
+  # -- The resource limits and requests assigned to script pod containers
+  # @section -- Script pod values
+  resources: 
+    requests:
+      memory: "100Mi"
+      cpu: "20m"
```

Comment:
Should the cpu limit be 25m to match the current default given by Tentacle?

Reply:
🤦 Yep, will fix